src/gosdt/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""
+Copyright (c) 2024 Ilias Karimalis. All rights reserved.
```

Comment:
Ilias, are we good to copy over the complete readme copyright info, for consistency? All rights reserved seems incompatible with our 3-clause license.